src/monitor/node_active_protocol.c

```diff
@@ -665,6 +675,24 @@ IsStateIn(ReplicationState state, List *allowedStates)
 }
 
 
+/* converts text representation of lsn to XlogRecPtr */
+static XLogRecPtr TextToLsn(char *lsnString)
```

Comment:
It might be that we can make it work without this function, which I think would be best.